assets/js/components/SelectionPanel/index.stories.js

```diff
@@ -128,6 +128,31 @@ WithSavedItems.scenario = {
 	label: 'Components/SelectionPanel/WithSavedItems',
 };
 
+export const withZeroUnsavedItems = Template.bind( {} );
+withZeroUnsavedItems.storyName = 'With zero unsaved items';
+withZeroUnsavedItems.args = {
+	availableSavedItems: Array.from(
+		{ length: 24 },
```

Comment:
It would be good to have less items in the lists for these stories, so we actually capture the "Additional items" group (or its absence) in the VRT images. I'd suggest replacing `24` for a much lower number (say, `5`) across the whole file, and updating the images for all the affected VRT scenarios.